doc/whats_new.rst

```diff
@@ -35,6 +35,8 @@ Enhancements
 - Some datasets out in the real world have a non-standard ``stim_type`` instead of a ``trial_type`` column in ``*_events.tsv``. :func:`mne_bids.read_raw_bids` now makes use of this column, and emits a warning, encouraging users to rename it, by `Richard Höchenberger`_ (:gh:`680`)
 - When reading data where the same event name or trial type refers to different event or trigger values, we will now create a hierarchical event name in the form of ``trial_type/value``, e.g. ``stimulus/110``  by `Richard Höchenberger`_ (:gh:`688`)
 - When reading data via :func:`mne_bids.read_raw_bids`, the channel names specified in the BIDS ``*_channels.tsv`` files now always take precedence (and do not need to match) the channel names stored in the raw files anymore, by `Richard Höchenberger`_ (:gh:`691`)
+- When reading data via :func:`mne_bids.read_raw_bids`, the channel names specified in the BIDS ``*_electrodes.tsv`` files now always take precedence (and do not need to match) the channel names stored in the raw files anymore, by `Adam Li`_ (:gh:`704`)
+- Add :func:`mne_bids.read_dig_bids` to allow reading of a ``DigMontage`` object directly from MEG/EEG/iEEG BIDS ``*electrodes.tsv`` and ``*coordsystem.json`` files, by `Adam Li`_ (:gh:`704`)
```

Comment:
just a note: I introduced a typo in https://github.com/mne-tools/mne-bids/commit/45f3f39c54472b3765692ef1fdfd285f1b6172f2, (missing a `)` in the changelog entry), please add that once you rebase here :) :) :)

Reply:
I also think lines 37 and 38 should be merged into a single entry

Reply:
and @eort requests an edit to .mailmap: https://github.com/mne-tools/mne-bids/commit/45f3f39c54472b3765692ef1fdfd285f1b6172f2#r47542622

Reply:
Done and done. lmk if it looks good.